lib/sentry-errors.js

```diff
@@ -46,14 +46,12 @@ const asyncTracker = Object.assign(new EventEmitter(), {
   }
 });
 
-exports.setup = function (options, dsn = process.env['SENTRY_DSN']) {
-  if (dsn) {
-    Raven.config(dsn, Object.assign({}, defaultOptions, options)).install();
-  }
+exports.setup = function (options) {
+  Sentry.init(Object.assign({}, defaultOptions, options))
   return exports;
 }
 
-exports.captureException = asyncTracker.trackMethod(Raven, 'captureException');
-exports.captureMessage = asyncTracker.trackMethod(Raven, 'captureMessage');
+exports.captureException = asyncTracker.trackMethod(Sentry, 'captureException');
+exports.captureMessage = asyncTracker.trackMethod(Sentry, 'captureMessage');
 exports.flush = () => asyncTracker.waitUntilInactive();
```

Comment:
It turns out the new SDK has built-in support for this! So you could delete all the `asyncTracker` code above and replace this line with:

```js
exports.captureException = Sentry.captureException;
exports.captureMessage = Sentry.captureMessage;
exports.flush = (timeout = maximumWaitTime) => Sentry.flush(timeout);
```

Docs: http://getsentry.github.io/sentry-javascript/modules/node.html#flush